### ISSUE TITLE
Question about pose alignment

### ISSUE THREAD

**Issue by trevor-taeyeop** [facebookresearch/meshrcnn#59]
Thanks for sharing the good work.

Given pointcloud(vertices or voxel), camera_intrinsic(K), pose(rotation and translation) from the annotations. 
I tired to check the pose alignment (project the pointcloud), But I couldn't get the accurate alignment.
(I checked based on your **"pix3d.py"**)

Could you advice which part are missing??
If there is an example related to this part, it would be a great help.

https://github.com/facebookresearch/meshrcnn/blob/ab0762f87bfd4cbdf4560982c5c33832a445380b/meshrcnn/data/datasets/pix3d.py#L177

```
  def project(xyz, K, RT):
        """
        xyz: [N, 3]
        K: [3, 3]
        RT: [3, 4]
        """
        xyz = np.dot(xyz, RT[:, :3].T) + RT[:, 3:].T
        xyz = np.dot(xyz, K.T)
        xy = xyz[:, :2] / xyz[:, 2:]
        return xy

   vis = draw_pix3d_dict(d, meta.thing_classes + ["0"])
        fpath = os.path.join(dirname, os.path.basename(d["file_name"]))
        #################################################
        # get point_cloud, pose, camera_intrinsic
        mesh_path = d['annotations'][0]['mesh']
        voxel_path = d['annotations'][0]['voxel']
        import meshrcnn.utils.shape as shape_utils
        voxel = shape_utils.read_voxel(voxel_path)
        vertices, faces = load_obj('/sdata1/workspace/meshrcnn/datasets/pix3d/'+mesh_path)
        focal = d['annotations'][0]['K'][0]
        cx = d['annotations'][0]['K'][1]
        cy = d['annotations'][0]['K'][2]
        R = np.array(d['annotations'][0]['R'])
        t = np.array(d['annotations'][0]['t'])[:,None]
        RT = np.concatenate((R,t),-1)
        K = np.array([[focal,0,cx],[0,focal,cy],[0,0,1]])
        ##################################################
        # projection
        xy = project(voxel[0].detach().cpu().numpy(), K, RT)
        ###################################################
        # visualization
        plth, pltw = 1,2
        plt.subplot(plth,pltw,1)
        plt.imshow(vis)
        plt.subplot(plth,pltw,2)
        plt.imshow(vis)
        plt.plot(xy[:,0],xy[:,1],'b.')
        plt.show()


```

![Screenshot from 2020-09-22 23-55-16](https://user-images.githubusercontent.com/50736858/93901185-373a1680-fd31-11ea-839b-2acf7d9a611d.png)

![Screenshot from 2020-09-23 00-30-35](https://user-images.githubusercontent.com/50736858/93903752-0c04f680-fd34-11ea-9b73-617920e2e04a.png)

**Comment by gkioxari** [comment]
Looks like your x- and y- axis need to be inverted.

You can check more about how we achieve alignment in the code and visualization code

https://github.com/facebookresearch/meshrcnn/blob/ab0762f87bfd4cbdf4560982c5c33832a445380b/meshrcnn/utils/vis.py#L224

To get these visualizations during training set `cfg.MODEL.VIS_MINIBATCH` to `True` during training.

**Comment by trevor-taeyeop** [comment]
Thanks for the quick and helpful advice.

**Comment by trevor-taeyeop** [comment]
I sovled my problem changing the camera intrinsic matrix

![Screenshot from 2020-09-23 13-38-44](https://user-images.githubusercontent.com/50736858/93967433-24135f00-fda2-11ea-8249-e6d61bfcb800.png)
